auth0/whitelist-github-teams.js

```diff
@@ -0,0 +1,32 @@
+function (user, context, callback) {
+  var request = require('request');
+  var github_team_whitelist = ['webops'];
```

Comment:
This works for our current requirements, but we'll need to refine it soon so that we give a lower level of access to regular users.